src/web/app/pages-help/instructor-help-page/instructor-help-section.component.ts

```diff
@@ -101,4 +101,12 @@ export class InstructorHelpSectionComponent implements OnInit, OnChanges, AfterV
       }
     }
   }
+
+  /**
+   * Checks if any question in the subsection is to be displayed after the search
+   */
+  displaySubsection(questionsToDisplay: Boolean[], firstPoint: number, lastPoint: number): boolean {
```

Comment:
Do you still need this now that #9395 is merged?

Reply:
No, I just had it left there in my own branch to keep the feature working. I'll remove it!